src/Common/MoistThermodynamics/MoistThermodynamics.jl

```diff
@@ -982,7 +983,7 @@ function saturation_adjustment(
     ρ::FT,
     q_tot::FT,
     maxiter::Int,
-    tol::FT,
+    tol::AbstractTolerance,
```

Comment:
It would be good to make clearer in the documentation above what this tolerance is precisely.